src/scss/base/_typography.scss

```diff
@@ -179,6 +179,13 @@
         }
     }
 
+    /**
+    * Buttons
+    */
+    button {
+        font-family: $font-family-base;
```

Comment:
Just out of interest – do we need this to override the browser default as it doesn't inherit from the font family set on the main document?

Reply:
I tested it and when the font for button is not specified it defaults back to browser default even with body font being ours. I didn't spent much time investigating and just added this style in to clear font from `o-btn` element and to still stay safe.

Reply:
I hit this recently in menu when working on the phase 1 icing changes where all buttons that didn't have the type-face set explicitly would revert back to use the browser default font (e.g. `sans` in Chrome).